.env.rtf

```diff
@@ -0,0 +1,13 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2578
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fnil\fcharset0 .AppleSystemUIFontMonospaced-Regular;}
+{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red255\green255\blue255;\red150\green204\blue255;
+}
+{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;\cssrgb\c100000\c100000\c100000;\cssrgb\c64706\c83922\c100000;
+}
+\paperw11900\paperh16840\margl1440\margr1440\vieww28600\viewh15140\viewkind0
+\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 url = \'93
+\f1 \cf2 \cb3 \expnd0\expndtw0\kerning0
+\outl0\strokewidth0 \strokec4 http://localhost:9000/uploadFile
+\f0 \cf0 \cb1 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 \'94}
```

Comment:
Why this?

Reply:
I dont know what that is but I’ll remove it now

Reply:
ahhh 
you want to hack zurichat through PR?🤣

Reply:
Lmao at all it's because of that .rtf extension that shouldn't be there

Reply:
Removed now